recipes/morphotree/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, CMake, tools
```

Comment:
please, use space instead of tabs for indentation https://peps.python.org/pep-0008/#tabs-or-spaces. Also, do not mix them. Observe linter warnings, it shows possible indentation errors.